MaeParserConfig.hpp

```diff
@@ -15,7 +15,7 @@
 #ifdef WIN32
 #define EXPORT_MAEPARSER __declspec(dllimport)
 #else
-#define EXPORT_MAEPARSER
+#define EXPORT_MAEPARSER __attribute__((visibility("default")))
```

Comment:
mind reordering so that it's 

    #ifdef WIN32
    #ifdef IN_MAEPARSER
    #define EXPORT_MAEPARSER __declspec(dllexport)
    #else
    #define EXPORT_MAEPARSER __declspec(dllimport)
    #endif

    #else
    #define EXPORT_MAEPARSER __attribute__((visibility("default")))
    #endif

Reply:
Done!